src/dq_suite/output_transformations.py

```diff
@@ -571,3 +572,37 @@ def write_validation_result_tables(
             table_name=validation_table_name,
             schema=schema,
         )
+
+
+def get_highest_severity_from_validation_result(validation_result: dict, rules_dict: dict) -> str | None:
```

Comment:
I think the return type is now only string, because `None` is no longer possible.